webapp/svelte.config.js

```diff
@@ -1,4 +1,4 @@
-import adapter from '@sveltejs/adapter-auto';
+import adapter from '@sveltejs/adapter-static';
```

Comment:
Can you pls explain why we are using adapter-static instead of adapter-auto or adapter-cloudflare?

Reply:
gotcha thank you for that